ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -64,9 +64,16 @@ def check(self, _):
         try:
             # Try to connect to check status of SSH
             try:
-                client.connect(
-                    self.host, port=self.port, username=self.username, password=self.password, pkey=private_key
-                )
+                if not private_key:
+                    client.connect(
+                        self.host, port=self.port, username=self.username, password=self.password, pkey=private_key
```

Comment:
If `private_key` evaluates False, why are we passing it in the connect statement?

Reply:
you're right, removing it